conf/keycloak/open-products-facts-realm.json

```diff
@@ -0,0 +1,2407 @@
+{
```

Comment:
I totally missed that. 😀 I'm not super sure about editing this config (yet), because I just created it by exporting the realm that I configured in Keycloak.